third_party/terraform/website/docs/r/container_cluster.html.markdown

```diff
@@ -493,6 +493,37 @@ maintenance_policy {
 
 In beta, one or the other of `recurring_window` and `daily_maintenance_window` is required if a `maintenance_policy` block is supplied.
 
+* `maintenance_exclusions_window` - (Optional) Exceptions to maintenance window. Non-emergency maintenance should not occur in these windows. A cluster can have up to three maintenance exclusions at a time [Maintenance Window and Exclusions](https://cloud.google.com/kubernetes-engine/docs/concepts/maintenance-windows-and-exclusions)
+
+Specify `start_time` and `end_time` in [RFC3339](https://www.ietf.org/rfc/rfc3339.txt) "Zulu" date format.  The start time's date is
```

Comment:
This format doesn't match others where we describe what a block supports, as seen below with:

The `ip_allocation_policy` block supports:

Reply:
`maintenance_exclusion` is child attribute of `maintenance_policy` which is already in the format as
 
The `maintenance_policy` block supports:

Should i need to add similar format for `maintenance_exclusion` as well ?

Reply:
Yeah I would. The current documentation of the fields is a bit confusing to me: https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/container_cluster#daily_maintenance_window

It doesn't specify the sub types in the same way as other objects on that page